.changeset/mighty-bulldogs-try.md

```diff
@@ -0,0 +1,34 @@
+---
+'gatsby-plugin-clerk': major
+'@clerk/chrome-extension': major
+'@clerk/nextjs': major
+'@clerk/clerk-react': major
+'@clerk/remix': major
+'@clerk/clerk-expo': major
+---
+
+- Introduce `@clerk/clerk-react/error` and `@clerk/clerk-react/internal` subpath exports to expose some internal utilities. Eg
+    ```typescript
+    // Before
+    import { __internal__setErrorThrowerOptions } from '@clerk/clerk-react';
+    // After
+    import { setErrorThrowerOptions } from '@clerk/clerk-react/internal';
+
+    // Before
+    import { isClerkAPIResponseError, isEmailLinkError, isKnownError, isMetamaskError } from '@clerk/clerk-react';
+    // After
+    import { isClerkAPIResponseError, isEmailLinkError, isKnownError, isMetamaskError } from '@clerk/clerk-react/error';
+
+    // Before
+    import { MultisessionAppSupport } from '@clerk/clerk-react';
+    // After
+    import { MultisessionAppSupport } from '@clerk/clerk-react/internal';
```

Comment:
❓ why are we moving this to `/internal`?

Reply:
> This is a component that handles the intermediate “undefined” state for multisession apps by unmounting and remounting the components during the session switch (setActive call) in order to solve theoretical edge-cases that can arise while switching sessions. It is not documented and as far as I know, no one has actually used it, however, we are currently using it in our dashboard. I suggest we either remove it or move it under /internal until we have more concrete use cases for this.

ref: `SDK Public API (v5)` RFC

Reply:
Makes sense! I was going to question why we were re-exporting this from other SDKs, but with the changes in 99207b29de2fe868e8b0ad200dc65c276df609bf this seems fine.